packages/portalnetwork/src/subprotocols/protocol.ts

```diff
@@ -132,6 +135,16 @@ export abstract class BaseProtocol {
         const pongMessage = decoded.value as PongMessage
         // Received a PONG message so node is reachable, add to routing table
         this.updateRoutingTable(enr, pongMessage.customPayload)
+        if (
+          this.client.discv5.sessionService.transport instanceof HybridTransportService &&
+          !this.client.discv5.sessionService.transport.webRTC?.peers.has(enr.nodeId) &&
+          enr.get('rtc') &&
+          toHexString(enr.get('rtc')!) === '0x01'
+        ) {
+          this.logger.extend('RTC')('Adding RTC conneection')
+          await this.client.discv5.sessionService.transport.connectWebRTC(enr.nodeId)
```

Comment:
Would it be possible to move this whole thing down to the transport layer somehow?  Ideally, the protocol layer stuff shouldn't need to worry about which transport layer we're using.  Could you move the `webRTC` maps up from `webRTC.ts` to `hybridTransport.ts` and then use the multiaddr or nodeId passed into `client.discv5.session.transport.send` to figure out whether to go with webRTC or websocket->UDP?

Reply:
Yes, in fact this was not necessary here at all, as it is happening in the transport itself now.  Deleting these lines should not have an affect